CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 - **cf-core:** Fix improper usage of `:not` to target non-nav `li` elements.
 - **cf-core:** Remove `margin-bottom` from last-child `li` elements.
 
+### Changed
+- [PATCH] **cf-layout:** Hero fixes:
+  - Use new breakpoint variables
+  - Subheading should be medium on smaller screens
```

Comment:
Related to other spec comment, lead-paragraph is _not_ medium, so either that DM page needs updating again, or this note is incorrect.